index.js

```diff
@@ -2275,5 +2283,6 @@
 //. [`fantasy-land/reduce`]:    v:fantasyland/fantasy-land#reduce-method
 //. [`fantasy-land/traverse`]:  v:fantasyland/fantasy-land#traverse-method
 //. [`fantasy-land/zero`]:      v:fantasyland/fantasy-land#zero-method
+//. [`Object.entries`]:         https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries
```

Comment:
I use case-sensitive ordering; running `:sort` in Vim does most of the work for me.

Please insert this entry between the `Traversable` and `` `fantasy-land/alt` `` entries. :)

Reply:
Ah, I tried sorting with vscode, and it completely changed the order of everything, so I assumed items were grouped.